curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c8d9a73965c0a26d7f634.md

```diff
@@ -0,0 +1,155 @@
+---
+id: 643c8d9a73965c0a26d7f634
+title: Step 28
+challengeType: 0
+dashedName: step-28
+---
+
+# --description--
+
+The `topic_list` object contains 4 more properties, but you only need the `topics` array. In this step, you will destructure the `topics` array from the `topic_list` object like this 
+
+```js
+const  { topics } = topic_list;
```

Comment:
I think we should remove the example here because campers will have had a lot practice with destructuring in earlier projects.

Reply:
done